app/main.py

```diff
@@ -1,3 +1,17 @@
 def cache(func):
-    # Write your code here
-    pass
+    cached_values = []
+
+    def inner(*args):
+
+        lst = [*args]
+
+        if lst in cached_values:
```

Comment:
`cached_values` is better to be a `dict`, because checking it item is in a dict is cheaper than checking if item in a list.